contributing.md

```diff
@@ -6,7 +6,13 @@ Start a new [Discussion](https://github.com/tendermint/starport/discussions/new)
 
 Thank you for your contribution!
 
-## Opening pull requests 
+## Feedback / Issues
```

Comment:
```suggestion
## Providing feedback
```

Reply:
@soniasingla (it's great to see you here, thanks for this PR!) let's follow the model of using active verbs for headings

Reply:
@barriebyron thanks for the great feedback, i have updated the PR, feel free to take a look! 🙌